src/pages/view-test-result/view-test-result.ts

```diff
@@ -56,6 +57,7 @@ export class ViewTestResultPage extends BasePageComponent implements OnInit, OnD
   loadingSpinner: Loading;
   subscription: Subscription;
   showErrorMessage: boolean = false;
+  testResult$ = new Observable<object | StandardCarTestCATBSchema>();
```

Comment:
Not sure why this is needed, can you explain?

Reply:
It's no longer needed. Removed in latest commit